website/docs/d/cloud_project_user_s3_credential.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "ovh"
+page_title: "OVH: cloud_project_user_s3_credential"
+sidebar_current: "docs-ovh-datasource-cloud-project-user-s3-credential"
+description: |-
+  Get the Secret Access Key of an Access Key ID of a public cloud project's user.
+---
+
+# ovh_cloud_project_user_s3_credential (Data Source)
+
+Use this data source to retrieve the Secret Access Key of an Access Key ID associated with a public cloud project's user.
+
+## Example Usage
+
+```hcl
+data "ovh_cloud_project_user_s3_credentials" "s3_credentials" {
+   service_name = "XXXXXX"
+   user_id      = "1234"
+}
+
+data "ovh_cloud_project_user_s3_credential" "s3_cred_1" {
```

Comment:
suggestion:

```
data "ovh_cloud_project_user_s3_credential" "mycred" {
   service_name = var.service_name
   user_id      = "244435"
   access_key_id = "xxxx"
}

output "myaccesskey" {
    value = ovh_cloud_project_user_s3_credential.s3_credential.access_key_id
}

output "mysecretaccesskey" {
    value = ovh_cloud_project_user_s3_credential.s3_credential.secret_access_key
    sensitive = true
}
```

or, if we want to use the datasource that list the access_key_id:

```
data "ovh_cloud_project_user_s3_credential" "mycred" {
   service_name = var.service_name
   user_id      = data.ovh_cloud_project_user_s3_credentials.myadmin.user_id
   access_key_id = data.ovh_cloud_project_user_s3_credentials.myadmin.access_key_ids[0]
}

output "myaccesskey" {
    value = ovh_cloud_project_user_s3_credential.s3_credential.access_key_id
}

output "mysecretaccesskey" {
    value = ovh_cloud_project_user_s3_credential.s3_credential.secret_access_key
    sensitive = true
}
```